framework/encode/custom_vulkan_api_call_encoders.cpp

```diff
@@ -403,6 +403,12 @@ VKAPI_ATTR uint64_t VKAPI_CALL GetBlockIndexGFXR()
     return manager->GetBlockIndex();
 }
 
+VKAPI_ATTR void VKAPI_CALL WriteAssetsGFXR()
```

Comment:
I believe you mentioned during our sync meeting that there was a private API that our layer could trigger to force an asset dump outside of a capture trigger, is this that function?

Reply:
Based on 87ce6507840794d59ded9fac887ae8e399327096, I'm pretty sure it is.

Reply:
Yes, `WriteAssetsGFXR` is exposed just like `GetBlockIndexGFXR` ( https://github.com/LunarG/gfxreconstruct/commit/87ce6507840794d59ded9fac887ae8e399327096#diff-4cd2c76addccf755b343693d32cb024f7b90d9d4d2fd2ecc7d997921ee1c533bL35 ) that you use in the perfetto layer and in the same fashion should it should be discoverable through the vulkan loader (I haven't tested it but crossing fingers that it should work)

Reply:
Realized that the naming I used was very inconsistent. Changed it to 
`{ "DumpAssetsGFXR", reinterpret_cast<PFN_vkVoidFunction>(encode::DumpAssetsGFXR) }`
Function should now be discoverable through the Vulkan loader by `vkGetInstanceProcAddr([instance], "DumpAssetsGFXR");`